uwblueprint/project-read-frontend
# Deleting member confirmation

### Notion ticket link



[Add a confirmation modal when users delete a child/gueste](https://www.notion.so/uwblueprintexecs/Add-a-confirmation-modal-when-users-delete-a-child-guest-1cde52f4ef85403c94331d589349cbd6)



### Implementation description

- Added confirmation modal when deleting members
- Added a close button to close the sidebar instead of clicking outside of it



### Steps to test

1. Verify deleting child/member from sidebar or form displays confirmation modal
2. Verify that both buttons on modal work as expected



### What should reviewers focus on?

- correctness

### Checklist
